src/vs/workbench/services/suggest/browser/simpleCompletionModel.ts

```diff
@@ -208,10 +208,33 @@ export class SimpleCompletionModel {
 				// Then by file extension length ascending
 				score = a.fileExtLow.length - b.fileExtLow.length;
 			}
-			if (score === 0 || fileExtScore(a.fileExtLow) === 0 && fileExtScore(b.fileExtLow) === 0) {
-				// both files or directories, sort alphabetically
-				score = a.completion.label.localeCompare(b.completion.label);
+			if (score === 0 || (fileExtScore(a.fileExtLow) === 0 && fileExtScore(b.fileExtLow) === 0)) {
```

Comment:
Should this check `a`/`b` `.isFile`/`.isDirectory` instead? I can't remember why `fileExtScore` is used here.

Reply:
I think this case is always true on mac - only true for directories on windows?

Reply:
Perhaps this condition should just be removed?

Reply:
I think it depends on how we want file/dir completion sorting should interact with other completions. This is what we do in insiders:

![image](https://github.com/user-attachments/assets/84615167-5a71-4f44-aebc-09ad3b764566)

Scores at 18, 18, 18, 18, 1, -2, -2, -2, -2, -2, -8, ...

Seems like the files should be above VSSVC.exe (score 1) in this case.